minetest/minetest
# Translate access denied strings

This PR allows access denied string to be translated. ~~Traditional Chinese translations are also added (for testing purposes, but they are usable strings).~~

~~Ignore the absolute mess in the `.po` files... `xgettext` reordered so many things.~~ Fixed in 3d4107598 and c844db993.

## To do

This PR is Ready for Review.

- [x] The `gettext` and the translation files
- [x] Clean up the mess of the diff... if that matters

## How to test

* Run Minetest with this PR with an English locale. Everything should work as normal.
* Apply the strings in https://github.com/minetest/minetest/pull/14842#issuecomment-2224198011, then run Minetest with this PR with a Traditional Chinese locale. The strings returned in the "access denied" screen should be translated.
* Run Minetest without this PR. Everything should work as if English is set.
